sdk/keyvault/tests.yml

```diff
@@ -10,13 +10,12 @@ resources:
 jobs:
   - template: ../../eng/pipelines/templates/jobs/archetype-sdk-tests.yml
     parameters:
-      MaxParallel: 1
+      MaxParallel: 5
```

Comment:
We should be able to collapse all these jobs down to a single job and remove `BuildTargetingString`

Reply:
@chlowell and I think it would be better to still leave them separated out bc pytest isn't working in the pipeline, and if we coalesced back, each entry in the matrix would take {time to run keys} + {time to run secrets} + {time to run certificates} length to run and we would run into timeouts before when we tried this. Now it basically takes max({time to run keys}, {time to run secrets}, {time to run certificates}) to run. If there's a resource issue (since this way takes more VMs), we can talk more about it too

Reply:
Makes sense then. From looking at the last scheduled run we'd have to set a timeout close to 3 hours.